website/sidebars.json

```diff
@@ -169,6 +169,7 @@
       "Monitor User Activity": [
         "tools/realtime",
         "tools/indexer-for-explorer",
+        "tools/indexing",
```

Comment:
maybe it would be better to put this in the `Community Tools` section?

Reply:
yes, you're right, Community Tools are a better fit for this kind of indexing tool list. 👍 I'll update the draft

Reply:
fixed